plugin/metrics/prometheus/options.go

```diff
@@ -127,10 +118,6 @@ func (opt *Options) InitFromViper(v *viper.Viper) error {
 	cfg.ConnectTimeout = v.GetDuration(cfg.namespace + suffixConnectTimeout)
 	cfg.TokenFilePath = v.GetString(cfg.namespace + suffixTokenFilePath)
 
-	cfg.SupportSpanmetricsConnector = v.GetBool(cfg.namespace + suffixSupportSpanmetricsConnector)
```

Comment:
Thanks, I also meant that we can remove the actual config (and all its uses): https://github.com/jaegertracing/jaeger/blob/5742846bf691f0c0a88e9259720973270ac0c426/pkg/prometheus/config/config.go#L31

Reply:
Done